homeassistant/components/philips_js/config_flow.py

```diff
@@ -178,6 +186,9 @@ async def async_step_init(
                     CONF_ALLOW_NOTIFY,
                     default=self.config_entry.options.get(CONF_ALLOW_NOTIFY),
                 ): bool,
+                vol.Optional(CONF_TURN_ON_ACTION): ActionSelector(
+                    ActionSelectorConfig()
```

Comment:
I think this is less powerful than custom integration triggers.I also don't like using option flows for automation settings. Automations should be configured separately from integrations that integrate devices and services.

Reply:
I see your point. I will look at the other pull again. Was not straight forward.